modules/ROOT/pages/installation/linux/debian.adoc

```diff
@@ -295,3 +295,16 @@ For more information, see xref:configuration/set-initial-password.adoc[].
 ====
 
 For more information on operating the Neo4j system service, see xref:installation/linux/systemd.adoc[Neo4j system service].
+
+== Uninstall Neo4j
+
+In case you want to specifically remove Neo4j Desktop, refer to https://neo4j.com/docs/desktop-manual/current/troubleshooting/re-install/[Neo4j Desktop -> Re-installation of Desktop].
```

Comment:
We don't need to mention Neo4j Desktop on each OS page, as there is nothing related to how to install Desktop on any of them. So, I think you should remove this sentence.